package/zmq_adapter/src/zmq_adapter.h

```diff
@@ -19,10 +19,12 @@
 
 #include <czmq.h>
 
-void debug_printf(const char *msg, ...);
 void io_loop_start(int read_fd, int write_fd);
 void io_loop_wait(void);
 void io_loop_wait_one(void);
 void io_loop_terminate(void);
 
+#define debug_printf(format, ...) \
```

Comment:
I created a macro so that I could actually add easily the calling function. There was no easy way with the static function and there were duplicated strings that were confusing (like "io loop begin", etc.)